lib/components/Body.js

```diff
@@ -50,6 +56,15 @@ var Body = React.createClass({
     if (isDate(body)) {
       return $(Data, null, body.toString());
     }
+    if (isSet(body)) {
+      var contents = Array.from(body)
```

Comment:
Oh I see, the only examples I could find involved `for..of`, so I figured a ponyfill would be easier.